modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/questions/Questions.es.js

```diff
@@ -587,7 +590,15 @@ export default withRouter(
 												}
 											);
 										}}
-										section={section}
+										parentSection={section.parentSection}
+										queryVariables={{
+											messageBoardSectionId: section.id,
+										}}
+										showTitle={true}
+										subscribeQuery={subscribeSectionQuery}
```

Comment:
I see... passing the queries without a real contract in the subscription APIs seems risky. Sending a function or something like that would be nicer.